CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Brew recipe links are now correctly made [#47](https://github.com/cyberark/terraform-provider-conjur/issues/47)
+- Made output binaries have a version suffix [#30](https://github.com/cyberark/terraform-provider-conjur/issues/30)
```

Comment:
are these clear enough of the value of each?

for example, maybe it would be clearer to say
> - Each brew recipe binary now includes the provider version
> - Updated output binary file names to include a version suffix so that the version command returns the correct version

or something - basically I'm asking if we're being clear enough about the outcome of each fix